### PR TITLE
Vote 351/reg dates ko

### PR DESCRIPTION
**Pull request by SamiraMSadat**

### CODE REVIEW

File: data/translations/ko/state-data.json [hunk 1]
```diff
@@ -3,310 +3,410 @@
...
     "more_info_link": "https://www.elections.il.gov/Default.aspx",
-    "ip_deadline": "선거당일까지 등록이 가능합니다.",
-    "online_deadline": "선거일 16일 전까지",
-    "mail_received_deadline": "반드시 선거일 28일 전까지 까지 접수해야 합니다.",
+    "ip_deadline": "2022년 11월 8일 화요일",
+    "online_deadline": "2022년 10월 18일 화요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 6일 목요일 까지 소인이 찍혀야 합니다.",
+    "default_ip_deadline": "선거당일까지 등록이 가능합니다.",
+    "default_online_deadline": "선거일 16일 전까지",
+    "default_mail_received_deadline": "반드시 선거일 28일 전까지 까지 접수해야 합니다.",
```
Comment (rayestrada) [inline data/translations/ko/state-data.json:182]: Update to `default_mail_postmarked_deadline`

File: data/translations/ko/state-data.json [hunk 2]
```diff
@@ -3,310 +3,410 @@
...
     "registration_link": "https://voter.svrs.nj.gov/register",
     "more_info_link": "https://nj.gov/state/elections/voter-registration.shtml",
-    "online_deadline": "선거일 21일 전까지",
-    "mail_received_deadline": "반드시 선거일 21일 전까지 까지 접수해야 합니다.",
-    "ip_deadline": "선거일 21일 전까지",
+    "ip_deadline": "2022년 10월 18일 화요일",
+    "online_deadline": "2022년 10월 18일 화요일",
+    "mail_postmarked_deadline": "반드시 2022년 10월 18일 화요일 까지 소인이 찍혀야 합니다.",
+    "default_online_deadline": "선거일 21일 전까지",
+    "default_mail_received_deadline": "반드시 선거일 21일 전까지 까지 접수해야 합니다.",
```
Comment (rayestrada) [inline data/translations/ko/state-data.json:372]: Updated to `default_mail_postmarked_deadline`

**Review by rayestrada — CHANGES_REQUESTED** [review]
@SamiraMSadat There are a couple minor revisions to be completed. Thanks.

File: data/translations/ko/state-data.json (hunk 1 above)
Comment (SamiraMSadat) [reply data/translations/ko/state-data.json:182]: fixed

File: data/translations/ko/state-data.json (hunk 2 above)
Comment (SamiraMSadat) [reply data/translations/ko/state-data.json:372]: fixed

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**